src/test/java/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,118 @@
+package hw2.ex1;
+
+import hw2.base.BaseClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Exercise1 extends BaseClass {
+
+    @Test
+    public void exerciseOneTest() {
+        SoftAssert sa = new SoftAssert();
+
+        // 1. Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+        String actualTitle = driver.getTitle();
+        String expectedTitle = "Home Page";
+
+        // 2. Assert Browser title
+        sa.assertEquals(actualTitle, expectedTitle);
+
+        // 3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        WebElement loginUserName = driver.findElement(By.id("user-name"));
+
+        // 4. Assert Username is loggined
+        sa.assertEquals(loginUserName.getText(), "ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+
+        List<WebElement> headerItems = driver.findElements(By
+                        .cssSelector("ul[class='uui-navigation nav navbar-nav m-l8'] > li"));
```

Comment:
Why do you specify all css classes for the current element?

Reply:
It is correct, every "li" item is found in this common class "uui-navigation nav navbar-nav m-l8". But I simplified to .m-l8 > li